core/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>amoro-ams-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
```

Comment:
Do we still need to declare the dependency here if the module `core` had dependent on `ams-api`.

Reply:
IIUC, we need to explict define the denpendency here, otherwise we can't find it when compiling. will double check here after has shaded all the jackson involded here